NSS-Day-Cohort-53/tabloid-full-stack-always-polite-iguanas
# user can edit a tag

#### Changes Made
1. Added file `editTag.js` to `tag` directory.
2. Modified file `ApplicationViews.js` to include `routing` and functionality.
3. Modified the `ITagRepository.cs` and `TagRepository.cs` to add methods to update a tag
​
close #32 

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout sr-edit-tag
    ```
4. Open a new Terminal tab (⌘T) and navigate to the server directory.
5. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user edits a tag, then user `should` be routed back to the list of tags with the updated tag 
6. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
